### DISCUSSION TITLE
Advanced options opens in gvim

### DISCUSSION THREAD

**Discussion by hme3** [sumatrapdfreader/sumatrapdf#3816] [General]
Advanced options opens in gvim which is not my default .txt editor. If I click on SumatraPDF-settings.txt it opens in default editor
v3.6.15800 64-bit

**Comment by GitHubRulesOK** [comment]
yes that was changed some time ago and there was an open issue , https://github.com/sumatrapdfreader/sumatrapdf/issues/2762
it is highly confusing when it always previously opened in native notepad

**Reply by hme3** [reply]
Documentation still says it should open with default .txt editor
